tools/gen_ort_dockerfile.py

```diff
@@ -70,33 +72,30 @@ def dockerfile_for_linux(output_file):
 # Install dependencies from
 # onnxruntime/dockerfiles/scripts/install_common_deps.sh. We don't run
 # that script directly because we don't want cmake installed from that
-# file.
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        wget \
-        zip \
-        ca-certificates \
-        build-essential \
-        cmake \
-        curl \
-        libcurl4-openssl-dev \
-        libssl-dev \
-        patchelf \
-        python3-dev \
-        python3-pip
+# file. 
+'''
+    # importing module and appending path
+    import sys
+    sys.path.append(FLAGS.common_repo_path + '/tools/')
```

Comment:
What is this code doing? Why do you need common/tools on the sys.path? Why did you change the way dependencies are installed?

Reply:
@deadeyegoodwin This is using https://github.com/triton-inference-server/common/pull/21 code to install all dependencies